py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -245,7 +245,40 @@ def aten_ops_fmod(
     return impl.elementwise.fmod(ctx, target, SourceIR.ATEN, name, args[0], args[1])
 
 
-@dynamo_tensorrt_converter(torch.ops.aten.relu.default)  # type: ignore[misc]
+@dynamo_tensorrt_converter(torch.ops.aten.grid_sampler)
+@dynamo_tensorrt_converter(torch.ops.aten.grid_sampler_2d.out)
+@dynamo_tensorrt_converter(torch.ops.aten.grid_sampler_2d_backward.out)
+@dynamo_tensorrt_converter(torch.ops.aten.grid_sampler_3d.out)
+@dynamo_tensorrt_converter(torch.ops.aten.grid_sampler_3d_backward.out)
+@enforce_tensor_types(
+    {
+        0: (TRTTensor,),
+        1: (TRTTensor,),
+    }
+)  # type: ignore[misc]
+def aten_ops_grid(
+    ctx: ConversionContext,
+    target: Target,
+    args: Tuple[Argument, ...],
+    kwargs: Dict[str, Argument],
+    name: str,
+) -> Union[TRTTensor, Sequence[TRTTensor]]:
+    return impl.grid.grid(
+        ctx, 
+        target, 
+        SourceIR.ATEN, 
+        name, 
+        input=args[0], 
+        grid=args[1], 
+        interpolation_mode=args[2], 
+        padding_mode=args[3], 
+        align_corners=args_bounds_check(args, 4, True),
+        output_mask=args_bounds_check(args, 5, None),
```

Comment:
This can be removed once the `out` variants are removed from the decorator list

Reply:
Do you mean the` output_mask` argument? As far as I understood, the `.backward` would have the `output_mask` argument.

Reply:
Yes, I meant the `output_mask` argument. I don't think we should ever encounter an argument of the `backward` variety since the backend is strictly forward/inference-only